src/clj/rems/css/styles.clj

```diff
@@ -88,11 +88,46 @@
    [".form-control:-ms-input-placeholder" {:color "#555"}])) ; Internet Explorer 10-11
 
 (defn- generate-media-queries []
+  (list
+   (stylesheet/at-media {:max-width (u/px 480)}
+                        [(s/descendant :.rems-table.cart :tr)
+                         {:border-bottom "none"}])
+   (stylesheet/at-media {:max-width (u/px 870)}
+                        [:.user-widget [:.icon-description {:display "none"}]])
+   (stylesheet/at-media {:min-width (u/px 480)}
+                        [:.commands {:white-space "nowrap"}])
+   (stylesheet/at-media {:prefers-reduced-motion :reduce}
+                        [:body {:scroll-behavior :auto}])))
+
+(defn- generate-logo-styles []
+  (list
+   [:.logo-menu {:height logo-height-menu
+                 :background-color (get-theme-attribute :logo-bgcolor)
+                 :width "100px"}]
+   [:.logo {:height logo-height
+            :background-color (get-theme-attribute :logo-bgcolor)
+            :width "100%"
+            :margin "0 auto"
+            :padding "0 20px"
+            :margin-bottom (u/em 1)}]
+   [(s/descendant :.logo :.img)
+    (s/descendant :.logo-menu :.img)
+    {:height "100%"
+     :background-color (get-theme-attribute :logo-bgcolor)
+     :-webkit-background-size :contain
+     :-moz-o-background-size :contain
+     :-o-background-size :contain
+     :background-size :contain
+     :background-repeat :no-repeat
+     :background-position [[:center :center]]
+     :background-origin (get-theme-attribute :logo-content-origin)}]
+   [(s/descendant :.logo :.img) {:background-image (get-logo-image context/*lang*)}]
+   [(s/descendant :.logo-menu :.img) {:background-image (get-navbar-logo context/*lang*)}]))
+
+(defn- generate-logo-media-queries []
```

Comment:
Do we need separate `generate-logo-media-queries` and `generate-logo-styles`? Could they just be one chunk since they're all related to the logos? Or is there some reason why we need to have `generate-logo-styles` early in the main stylesheet and `generate-logo-media-queries` late?

Reply:
It is considered to be a good practice to separate media rules into separate mixins, in our case: functions

Reply:
I think these are two separate things. The articles you linked suggest putting the media queries into the component i.e. logo here. They suggest writing those media queries with reusable helpers and not by hand. What you are doing here is not creating a reusable media query mixing, you are simply splitting the code into two places.

Reply:
I suggest that you put the logo related stuff including styles and media queries into one function `generate-logo-styles` and remove the `generate-logo-media-queries. We can later refactor the media queries themselves.